### DISCUSSION TITLE
How to use Weblate to contribute on repositories I don't have access to?

### DISCUSSION THREAD

**Discussion by mokazemi** [WeblateOrg/weblate#9327] [Q&A]
Hi. I want to use Weblate to contribute and maintain a project (Jitsi-meet in this case) translations in my language.

I **forked** the repository in my Github, then **created a project** in my Weblate account and added a component.
I set  Version control system to **Git**,   Source code repository to **My fork**,  Repository branch to **Master**,  Repository push URL to **My fork plus https token and username**, and push branch to a branch called "fa-translation". Then I should go manually to My fork and pull request to the main project whenever I want. Also sync my fork with upstream repo.

Is it the correct way to do it? 
Also I wanted to ask what does "Github with pull request" option do?

**Comment by nijel** [comment]
That sounds correct.

As for the pull request, see https://docs.weblate.org/en/latest/admin/continuous.html#pushing-changes-from-weblate

**Reply by mokazemi** [reply]
Thanks. But I have a little issue here. I'm pulling from Master but pushing to fa-translation branch. So, doesn't it make my translations disappear after every pull/sync?

Also honestly didn't got the use case of pull request option. Is it for projects with protected master branch for example? 🤔

**Reply by nijel** [reply]
Weblate will merge or rebase them, they won't disappear.
